README.md

```diff
@@ -1,2 +1,21 @@
+![Logo](Resources/logo-black-transparent.png)
 # Aquarius
-Contains the source code for Aquarius, a 2D game engine for the ECE 5010 junior software design project
+A 2D game engine developed by Colton Smith, Daniel Burke, & Liam Warren for the Junior Software Design Project (ECE 5010).
+
+## Additional Required Software
+- CMake Version 3.10 ([Download](https://cmake.org/download/))
+
+## Building Aquarius
+1. Unzip the ZIP File
+2. Navigate in command-line interface to the unzipped project folder 
+3. Make a build directory by executing the following command `mkdir build`
+4. Navigate to the build directory by executing the following command `cd build`
+5. Execute CMake to build the project by executing the following command `cmake ..`
+6. Generate an executable file by executing the following command `cmake --build . --config Release`
```

Comment:
I usually don't use the cmake command for building, but I wonder what this does on windows (I know on Linux and MacOS it just invokes make), either way it is nice that we do this in a platform-agnostic way (can't assume everyone is using make lol)

Reply:
Isnt make used in windows for building cmake projects?

Reply:
ehhh it can be, but say you ran `cmake ..` with no args, it generates Visual Studio sln files. Like CLion comes with a version of make for windows so that's what it uses, and i'm pretty sure git for windows also comes with some sort of minGW make tool as well

Reply:
AH so when you invoke cmake --build it will use a platform specific build tool. In the case of windows, it uses MSBuild (same as UE4)